app/javascript/admin/controllers/image_upload_controller.js

```diff
@@ -35,11 +35,11 @@ export default class ImageUploadController extends Controller {
   }
 
   onUploadSuccess(result) {
-    this.imageResultTarget.classList.remove('d-none');
+    this.imageResultTarget.classList.remove('hidden');
     const output = `
-      <div class="form-group">
-        <label for="output">Image URL:</label>
-        <textfield id="output" name="output" class="form-control" readonly>
+      <div>
```

Comment:
**Non-blocking suggestion:** After an upload the image overlays a bit the textfield. A small tweak like this would make it align just a bit better

```suggestion
      <div class="mb-4">
```

**Before**

<img width="400" alt="Screen Shot 2022-12-21 at 09 06 43" src="https://user-images.githubusercontent.com/6045239/208937656-263830c5-88c1-487e-9927-85b0f1fde8cd.png">

**After**

<img width="400" alt="Screen Shot 2022-12-21 at 09 10 30" src="https://user-images.githubusercontent.com/6045239/208937753-7c24a124-6732-4735-9049-c415c2083fe4.png">

Reply:
Thanks